handsome-hedgehogs/workforce-management
# Fixinitializer

## Status
Choose:
**READY**

## Description 
Fixed initializer to correct order of element creation.  Also added some missing required fields

## Steps to Test or Reproduce
1. git fetch
2. git checkout fixinitializer
3. run solution
4. confirm *site/employees produces table
4. confirm *site/departments produces table
4. confirm *site/trainingprogramsproduces table
4. confirm *site/computers produces table

## Impacted Areas in Application
startup.cs
